packages/app/src/cli/constants.ts

```diff
@@ -225,6 +236,7 @@ export const externalExtensionTypeNames = {
     'Post-purchase UI',
     'Theme app extension',
     'Checkout UI',
+    'Checkout UI Beta',
```

Comment:
This should be UI Extension no?

Reply:
No I don't think so.

This is the name we present to users.  If we use UI Extension, how might we distinguish between options for other types of UI extensions? E.g: Post Purchase, Subscription or Customer accounts?  Yes, eventually these will all be a ui_extension under the hood, but we still need a way to group extension points into pre defined templates that roughly map to concepts like post purchase, subscription or customer accounts.  Which is what we are doing here.

Does that make sense? What do you think?